examples/governance_policy.py

```
@@ -0,0 +1,28 @@
+
+
+class DenyIfContainsSecret(GovernancePolicy):
+    """Example governance policy: deny when 'secret' appears in input text."""
+
+    def __init__(self, keyword: str = "secret") -> None:
+        self.keyword = keyword.lower()
+
+    async def evaluate(
+        self,
+        *,
+        core: object,
```

Ruff (0.14.7)

Unused method argument: `core`

(ARG002)